src/ontology/mondo-edit.obo

```diff
@@ -158072,15 +158073,15 @@ xref: MedDRA:10052181 {source="Orphanet:270", source="Orphanet:270/e"}
 xref: MESH:D039141 {source="MONDO:equivalentTo", source="Orphanet:270", source="DOID:11719", source="Orphanet:270/e"}
 xref: NCIT:C84942 {source="MONDO:equivalentTo", source="DOID:11719"}
 xref: NORD:1523 {source="MONDO:NORD"}
-xref: OMIM:164300 {source="MONDO:equivalentTo", source="Orphanet:270", source="DOID:11719", source="Orphanet:270/e"}
+xref: OMIMPS:164300 {source="MONDO:equivalentTo"}
 xref: Orphanet:270 {source="MONDO:equivalentTo", source="OMIM:164300"}
 xref: SCTID:77097004 {source="MONDO:equivalentTo", source="DOID:11719"}
 xref: UMLS:C0244252 {source="NCIT:C84942", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C0270952 {source="MONDO:equivalentTo", source="Orphanet:270", source="MONDO:ncbi_mim2gene_medline", source="OMIM:164300", source="DOID:11719", source="Orphanet:270/e"}
 is_a: MONDO:0004746 {source="DOID:11719"} ! myopathy of extraocular muscle
 is_a: MONDO:0016106 {source="Orphanet:270"} ! progressive muscular dystrophy
 is_a: MONDO:0020121 {source="DOID:11719", source="MESH:D039141", source="MONDO:Redundant", source="NCIT:C84942", source="Orphanet:270/inferred"} ! muscular dystrophy
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/8565 {source="MONDO:mim2gene_medgen"} ! PABPN1
```

Comment:
I think this term needs to be split b/c you are changing the meaning here

Reply:
in this case, if we remove the axiom here, this changes the meaning of this term though, right?